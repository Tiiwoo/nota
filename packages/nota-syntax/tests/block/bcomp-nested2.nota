@h1:
  Hello

  @span:
    world
  
  @span: yed

not-in-block