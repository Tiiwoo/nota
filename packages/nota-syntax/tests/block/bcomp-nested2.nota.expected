el(
  "h1",
  {},
  el("p", {}, "Hello"),
  el("span", {}, "world"),
  el("span", {}, "yed")
);